utils/swaggerDefinition.js

```diff
@@ -248,6 +248,14 @@ const swaggerOptions = {
             }
           }
         },
+        userAvailable: {
+          type: 'object',
+          properties: {
```

Comment:
Conflicts with the actual response sent by the route.
Is there any contract written for this API?
What is the expected response?